packages/workspace/src/workspace/state/in_mem.ts

```diff
@@ -71,6 +75,78 @@ export const buildInMemState = (
     await (await stateData()).elements.delete(id)
   }
 
+  const updateAccounts = async (accounts?: string[]): Promise<void> => {
+    const data = await stateData()
+    const newAccounts = accounts ?? await awu(getUpdateDate(data).keys()).toArray()
+    return getUpdateDate(data).setAll(
+      awu(newAccounts.map(s => ({ key: s, value: new Date(Date.now()) })))
```

Comment:
`awu` not needed here
```suggestion
      newAccounts.map(s => ({ key: s, value: new Date(Date.now()) }))
```